test/e2e/network_create_test.go

```diff
@@ -375,4 +375,19 @@ var _ = Describe("Podman network create", func() {
 		Expect(nc).To(ExitWithError())
 	})
 
+	It("podman network create with internal should not have dnsname", func() {
+		net := "internal-test" + stringid.GenerateNonCryptoID()
+		nc := podmanTest.Podman([]string{"network", "create", "--internal", net})
+		nc.WaitWithDefaultTimeout()
+		defer podmanTest.removeCNINetwork(net)
+		Expect(nc.ExitCode()).To(BeZero())
+		found, _ := nc.ErrorGrepString("dnsname and --internal networks are incompatible")
+		Expect(found).To(BeTrue())
```

Comment:
Please change this to `Expect(nc.ErrorToString()).ToNot(ContainSubstring("dnsname and --internal networks are incompatible"))`. This creates a much better error message with ginkgo.

Reply:
fixed